From dc3fb561439bff0dc6e410b09ed18203640bc5db Mon Sep 17 00:00:00 2001
From: Renato Araujo Oliveira Filho <renato.filho@canonical.com>
Date: Wed, 16 Dec 2015 16:02:16 -0300
Subject: [PATCH] Add missing include.

---
 libbuteosyncfw/profile/ProfileManager.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libbuteosyncfw/profile/ProfileManager.h b/libbuteosyncfw/profile/ProfileManager.h
index 87b23ee..e852db1 100644
--- a/libbuteosyncfw/profile/ProfileManager.h
+++ b/libbuteosyncfw/profile/ProfileManager.h
@@ -26,6 +26,7 @@
 
 #include "SyncProfile.h"
 #include "Profile.h"
+#include <QObject>
 #include <QList>
 #include <QHash>
 
-- 
2.1.4

