=== modified file 'libbuteosyncfw/libbuteosyncfw.pro'
--- old/libbuteosyncfw/libbuteosyncfw.pro	2015-07-09 20:09:37 +0000
+++ new/libbuteosyncfw/libbuteosyncfw.pro	2015-07-10 15:49:12 +0000
@@ -99,7 +99,7 @@
 QMAKE_CLEAN += lib$${TARGET}.prl pkgconfig/*
 
 # install
-target.path = /usr/lib/
+target.path  = $${LIBDIR}
 
 equals(QT_MAJOR_VERSION, 4): headers.path = /usr/include/buteosyncfw
 equals(QT_MAJOR_VERSION, 5): headers.path = /usr/include/buteosyncfw5

